.github/workflows/alpha-release.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '20.x'
+          node-version: 'lts/*'
```

Comment:
should we move this to be a Github variable like the ubuntu version in case we need to change it?

Reply:
We could but we already used lts in some of the pipelines. 

One of the advantages of using LTS is that the Node.js version will automatically migrate as LTS roles change. This ensures that the version won't be forgotten, reducing the need for manual updates. Also node is usually pretty good with backward compatibility (and deprecation policies)